scripts/build.sh

```diff
@@ -0,0 +1 @@
+NODE_ENV=production yarn install && yarn build
```

Comment:
once this PR merges, we can update fleek mainnet and rinkeby to use this script for building. Means we have a single source of truth.